closure/goog/testing/testsuite.js

```diff
@@ -27,13 +27,13 @@ var TestSuiteOptions;
  * the given object. For use in tests that are written as JavaScript modules
  * or goog.modules.
  *
- * @param {!Object<string, function()|!Object>} obj An object with one or more
- *     test methods, and optional setUp, tearDown and getTestName methods. The
- *     object may also have nested Objects (named like tests, i.e.
- *     `testNestedSuite: {}`) that will be treated as nested testSuites. Any
- *     additional setUp will run after parent setUps, any additional tearDown
- *     will run before parent tearDowns. The this object refers to the object
- *     that the functions were defined on, not the full testSuite object.
+ * @param {!Object} obj An object with one or more test methods, and optional
```

Comment:
This is its own separate change, caused by the two repos being out of sync

Reply:
fixed by passing base CL as a bare arg to copybara